### PR TITLE
Release notes for v1.16.0

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+  key. See :ref:`projections`.
+* Added the method :class:`~event_model.DocumentRouter.emit` and an ``emit``
+  parameter to :class:`~event_model.DocumentRouter` to support chaining them.
+* The :class:`~event_model.Filler` now provides public methods for clearing its caches,
+  :meth:`~event_model.Filler.clear_handler_cache` and
+  :meth:`~event_model.Filler.clear_document_caches`.
+* The method :meth:`~event_model.Filler.deregister_handler` returns the handler
+  that it has deregistered.
+* The ``filler_state`` attribute of :class:`~event_model.Filler` now includes
+  attributes ``resource`` and ``datum`` which may be used by coersion functions
```
Comment (cryos) [inline docs/source/release-history.rst:21]: s/coersion/coercion

File: docs/source/data-model.rst [hunk 2]
```diff
@@ -139,13 +139,19 @@ A typical example
...
-It is intended that a projection is an aid to interacting with external systems using standardized vocabularies.
-Projections might be used in a variety of use cases such as providing run data to analysis tools or suitcases.
-Each projection represents multiple ways to represent data from the run. Each field in the projection dictionary 
-is an unique and externally-identifiable string and each value is an instruction for accessing dat from the run. 
+.. _projections:
+
+Projections (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Run Start document may include a ``projections`` field.  It is intended
```
Comment (cryos) [inline docs/source/data-model.rst:147]: Two spaces used after periods in this paragraph, consider 's/  / '

**Review by cryos — CHANGES_REQUESTED** [review]
A typo and a suggestion, looks good otherwise

**Review by MikeHart85 — APPROVED** [review]

**Review by cryos — APPROVED** [review]
Looks good to me

File: docs/source/data-model.rst (hunk 2 above)
Comment (mrakitin) [reply docs/source/data-model.rst]: @tacaswell has a long explanation of double blank spaces. I think it was discussed during one of the Pilot calls.

Comment (mrakitin) [reply docs/source/data-model.rst]: https://en.wikipedia.org/wiki/Sentence_spacing_in_language_and_style_guides 🤯

Comment (danielballan) [reply docs/source/data-model.rst]: https://www.cultofpedagogy.com/two-spaces-after-period/

Comment (cryos) [reply docs/source/data-model.rst]: I would argue that "Nothing Says Over 40 Like Two Spaces after a Period!" should be corrected to "Nothing Says Over 
50 Like Two Spaces after a Period!" 😆

**Pull request merged**